__test__/events.test.ts

```diff
@@ -0,0 +1,163 @@
+import eventHandler from "../pages/api/events/class";
+import { client } from "../lib/db";
+import { makeEventHTTPRequest, makeHTTPRequest } from "./__testutils__/testutils.test";
+import { CreateClassEvent, ClassEvent } from "../models/events";
+import { StatusCodes } from "http-status-codes";
+import RRule from "rrule";
+
+const FIELDS_NOT_ENTERED_CORRECTLY = "Fields are not correctly entered";
+
+let rule: string;
+
+beforeAll(async () => {
+  await client.query("DELETE from commitments");
+  await client.query("DELETE from calender_information");
```

Comment:
I don't know if this is worth fixing because it's in a bunch of spots but calendar is spelled wrong lolll

Reply:
I knew it lol I kept on spelling it as calendar but I just thought I was wrong lmao

Reply:
Yeah we should probably fix it btw. I know it sucks but lets make sure that we have the highest quality of work :)